code/modules/client/preference_setup/virgo/01_spriteaccessories.dm

```diff
@@ -0,0 +1,298 @@
+// Global stuff that will put us on the map
```

Comment:
What parts of this are used?

Reply:
All parts, but not all are 'used' for non-event species.